diff -Nurp x11-xserver-utils-7.4+2ubuntu3/xset/xset.c working/xset/xset.c
--- x11-xserver-utils-7.4+2ubuntu3/xset/xset.c	2009-02-01 05:04:01.000000000 -0800
+++ working/xset/xset.c	2010-03-15 19:02:50.000000000 -0700
@@ -1554,8 +1554,8 @@ usage(char *fmt, ...)
 #ifdef FONTCACHE
     fprintf(stderr, "    To control font cache:\n");
     fprintf(stderr, "\t fc [hi-mark [low-mark [balance]]]\n");
-    fprintf(stderr, "\t    both mark values spcecified in KB\n");
-    fprintf(stderr, "\t    balance value spcecified in percent (10 - 90)\n");
+    fprintf(stderr, "\t    both mark values specified in KB\n");
+    fprintf(stderr, "\t    balance value specified in percent (10 - 90)\n");
     fprintf(stderr, "    Show font cache statistics:\n");
     fprintf(stderr, "\t fc s\n");
 #endif
